scripts/globals/pets/avatar.lua

```diff
@@ -136,6 +139,28 @@ xi.pets.avatar.onMobSpawn = function(pet)
                 petArg:setLocalVar(buffModeVar, 1)
             end
         end)
+    elseif pet:getPetID() == xi.petId.ALEXANDER then
+        pet:setLocalVar(alexanderSummonTimeVar, os.time())
+        pet:setSpeed(0)
```

Comment:
I was originally looking at using the mob mod `NO_MOVE` but that would also require adding a check in `CPetController::DoRoamTick` which seems overkill for a single pet.